src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -998,7 +1011,7 @@ namespace AppInstaller::CLI::Workflow
                 OpenSource() <<
                 SearchSourceForSingle <<
                 HandleSearchResultFailures <<
-                EnsureOneMatchFromSearchResult(false) <<
+                EnsureOneMatchFromSearchResult(SearchPurpose::Show) <<

```

Comment:
This seems hacky

I guess this else block is only reachable in Show flow now? If that's the case, the else block should be moved to ShowCommand.cpp , and this GetManifest Workflow task could be removed.

If it's too much work,  please add a todo and comment saying it's only reachable in Show flow and should be removed .

Reply:
> If it's too much work, please add a todo and comment saying it's only reachable in Show flow and should be removed.

I took a stab at this, hope I did it right;